sources/api/bootstrap-containers/src/main.rs

```diff
@@ -87,21 +87,15 @@ use std::fs;
 use std::path::Path;
 use std::process::{self, Command};
 use std::str::FromStr;
+use constants;
```

Comment:
I would probably import the constant names instead of qualifying them everywhere. (Same in all places...)

```suggestion
use constants::{API_SOCKET, HOST_CTR_BIN, SYSTEMCTL_BIN};
```

Reply:
I think this is a case where the opposite is justifiable - qualified constants make it clear that they're not crate-specific, where unqualified ones look like "standard" constants you'd expect in the same file.

Reply:
> qualified constants make it clear that they're not crate-specific, where unqualified ones look like "standard" constants you'd expect in the same file.

I guess I don't have the expectation that constants are in the same file. To me they're like all other symbols. That is, I have to jump to definition to know where they are. Not a big deal though.